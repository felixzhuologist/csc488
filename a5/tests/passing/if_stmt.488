{
    var x, y : Boolean
    x := true
    if x then {
        x := false
        write "true1", newline
    } else {
        x := true
        write "false1", newline
    }
    write "after1", newline

    y := true
    if x and y then
        write "true2", newline
    else
        write "false2", newline
    write "after2", newline

}