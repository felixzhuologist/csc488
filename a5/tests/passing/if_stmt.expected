true1
after1
false2
after2
